### PR TITLE
Migrate to Create React App template.

### PR DESCRIPTION
**Pull request by dasuni-30**

### CODE REVIEW

File: petcare-sample/b2c/web-app/petdesk/web/react/public/config.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  baseUrl: "",
+  clientID: "",
+  signInRedirectURL: "http://localhost:3000",
+  signOutRedirectURL: "http://localhost:3000",
+  myAccountAppURL: "",
+  resourceServerURL: "http://localhost:9090",
+  billingServerURL: "http://localhost:9091",
+  scope: ["openid", "email", "profile"],
+  enableOIDCSessionManagement: true
+};
```
Comment (brionmario) [inline petcare-sample/b2c/web-app/petdesk/web/react/public/config.js:11]:
```suggestion
};

```

File: petcare-sample/b2c/web-app/petdesk/web/react/src/App.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    <Router>
+    <Routes>
+        <Route path="/" element={ <HomePage /> } />
+        <Route element={ <NotFoundPage /> } />
+    </Routes>
+</Router>
+  );
+}
+
+export default App;
```
Comment (brionmario) [inline petcare-sample/b2c/web-app/petdesk/web/react/src/App.tsx:36]:
```suggestion
export default App;

```

File: petcare-sample/b2c/web-app/petdesk/web/react/src/components/GetThumbnail/get-thumbnail.ts [hunk 3]
```diff
@@ -6,9 +6,15 @@ export async function getThumbnail(accessToken: string, petId: string) {
...
     const response = await getPetInstance().get("/pets/" + petId + "/thumbnail" , {
       headers: headers,
       responseType: 'blob',
     });
     return response;
-  }
\ No newline at end of file
+  } catch (error) {
+    // Handle errors here
+    console.error('Error fetching thumbnail:', error);
```
Comment (brionmario) [inline petcare-sample/b2c/web-app/petdesk/web/react/src/components/GetThumbnail/get-thumbnail.ts:17]: Remove these logs.
Add a comment if you are making the errors silent.

File: petcare-sample/b2c/web-app/petdesk/web/react/src/components/UserMenu.tsx [hunk 4]
```diff
@@ -100,7 +100,7 @@ export default function MenuListComposition(props: {
                 open={userMenuOpen}
                 onClose={handleClose}
                 className="menu-style"
-                getContentAnchorEl={null}
+                // getContentAnchorEl={null}
```
Comment (brionmario) [inline petcare-sample/b2c/web-app/petdesk/web/react/src/components/UserMenu.tsx:103]: ```suggestion```

Comment (brionmario) [inline petcare-sample/b2c/web-app/petdesk/web/react/src/components/UserMenu.tsx:103]:
```suggestion
```

File: petcare-sample/b2c/web-app/petdesk/web/react/public/config.js (hunk 1 above)
Comment (dasuni-30) [reply petcare-sample/b2c/web-app/petdesk/web/react/public/config.js]: Added with 566218d543bee0fe0230971d4e69d3566fa32f65.

File: petcare-sample/b2c/web-app/petdesk/web/react/src/app.tsx [hunk 5]
```diff
@@ -16,35 +16,21 @@
...
+        <Route path="/" element={ <HomePage /> } />
+        <Route element={ <NotFoundPage /> } />
+    </Routes>
+</Router>
+  );
+}
 
-const root = createRoot(document.getElementById("root"));
-root.render(<App />);
+export default App;
```
[the remark this answers is not in the document]
Comment (dasuni-30) [reply petcare-sample/b2c/web-app/petdesk/web/react/src/app.tsx]: Added with 566218d543bee0fe0230971d4e69d3566fa32f65.

File: petcare-sample/b2c/web-app/petdesk/web/react/src/components/UserMenu.tsx (hunk 4 above)
Comment (dasuni-30) [reply petcare-sample/b2c/web-app/petdesk/web/react/src/components/UserMenu.tsx]: Removed with 566218d543bee0fe0230971d4e69d3566fa32f65.

File: petcare-sample/b2c/web-app/petdesk/web/react/src/components/GetThumbnail/get-thumbnail.ts (hunk 3 above)
Comment (dasuni-30) [reply petcare-sample/b2c/web-app/petdesk/web/react/src/components/GetThumbnail/get-thumbnail.ts]: Removed with 566218d543bee0fe0230971d4e69d3566fa32f65.

File: petcare-sample/b2c/web-app/petdesk/web/react/src/App.tsx (hunk 2 above)
Comment (dasuni-30) [reply petcare-sample/b2c/web-app/petdesk/web/react/src/App.tsx]: Removed with Added with 566218d543bee0fe0230971d4e69d3566fa32f65..

File: petcare-sample/b2c/web-app/petdesk/web/react/src/pages/home.tsx [hunk 6]
```diff
@@ -207,157 +207,157 @@ export const HomePage: FunctionComponent = (): ReactElement => {
...
+                        pet={pet}
+                        petThumbnail={thumbnail}
+                        setPetThumbnail={setThumbnail} />
+                    </div>
+                </>
+                )
+            }
+            </DefaultLayout>
+        );
 };
```
Comment (brionmario) [inline petcare-sample/b2c/web-app/petdesk/web/react/src/pages/home.tsx:363]:
```suggestion
};

```

**Review by brionmario — APPROVED** [review]

**Pull request merged**